python/lsst/meas/extensions/gaap/_gaussianizePsf.py

```diff
@@ -119,7 +119,7 @@ def run(self, exposure: lsst.afw.image.Exposure, center: lsst.geom.Point2D,  # n
         kernelCellSet = result.kernelCellSet
         targetPsfModel = result.targetPsfModel
         fwhmScience = exposure.getPsf().computeShape(center).getDeterminantRadius()*sigma2fwhm
-        fwhmModel = targetPsfModel.computeShape().getDeterminantRadius()*sigma2fwhm
+        fwhmModel = targetPsfModel.getSigma()*sigma2fwhm
```

Comment:
Ah, this works because it's always a `GaussianPsf`?  I think you should add a comment to that effect.